src/status_im2/integration_test/core.cljs

```diff
@@ -0,0 +1,58 @@
+(ns status-im2.integration-test.core
+  (:require
+    [cljs.test :refer [deftest is]]
+    [day8.re-frame.test :as rf-test]
+    [re-frame.core :as rf]
+    [status-im.events]
+    [status-im.multiaccounts.logout.core :as logout]                                   ; so integration tests can run independently
+    [status-im.subs.root]
+    [status-im.utils.test :as utils.test]
+    [status-im2.events]
+    [status-im2.integration-test.chat]
+    [status-im2.integration-test.constants :as constants]
+    [status-im2.integration-test.helpers :as helpers]
+    [status-im2.integration-test.wallet]
+    [status-im2.navigation.core]                                  ; so integration tests can run independently
+    [status-im2.subs.root]
+    [taoensso.timbre :as log])
+  (:require-macros
+    [status-im2.integration-test.macros :refer [with-app-initialized with-account]]))
+
+(utils.test/init!)
+
+(deftest initialize-app-test
+  (log/info "========= initialize-app-test ==================")
+  (rf-test/run-test-async
+    (rf/dispatch [:app-started])
+    (rf-test/wait-for
+      ;; use initialize-view because it has the longest avg. time and
+      ;; is dispatched by initialize-multiaccounts (last non-view event)
+      [:profile/get-profiles-overview-success]
+      (rf-test/wait-for
+        [:font/init-font-file-for-initials-avatar]
+        (helpers/assert-app-initialized)))))
+
+(deftest create-account-test
+  (log/info "====== create-account-test ==================")
+  (rf-test/run-test-async
+    (with-app-initialized
+      (with-account
+        (helpers/assert-messenger-started)
+        (helpers/logout!)
+        (rf-test/wait-for [::logout/logout-method])))))
+
+(deftest create-community-test
```

Comment:
Maybe better is to extract this community test outside the `core` namespace, and into `integration_test/community.cljs`, just like you did for `wallet.cljs` and `chat.cljs`.